druid/hadoop/common.runtime.properties

```diff
@@ -34,3 +34,6 @@ druid.storage.storageDirectory=var/druid/segments
 # For local disk (only viable in a cluster if this is a network mount):
 druid.indexer.logs.type=file
 druid.indexer.logs.directory=/tmp/druid/indexing-logs
+
+# Add extensions
+druid.extensions.loadList=["druid-datasketches"]
```

Comment:
@jamedge please add it to 

https://github.com/pubnative/docker-images/blob/53cd32f467f262bb0c3c703c09a231b74815b6cf/druid/hadoop/Dockerfile#L33

to keep all extensions in one place

Reply:
👍 I just confirmed that we use `druid-base` images for middle managers and `manager-hot`, so I'll add this extension there.

Reply:
So, as we discussed, this image will be used, with extension pulled manually as it's for the others as well :)